### ISSUE TITLE
[Mellanox] testQosSaiDwrr failed on Mellanox platform

### ISSUE THREAD

**Issue by bingwang-ms** [sonic-net/sonic-mgmt#10299]
**Description**
The `testQosSaiDwrr` test began to fail after PR https://github.com/sonic-net/sonic-mgmt/pull/9587.
It's because we disable TX by changing the scheduler. That caused impact to the DWRR test.

I tried to revert the change, the test can pass.

[testQosSaiDwrr.log](https://github.com/sonic-net/sonic-mgmt/files/12873575/testQosSaiDwrr.log)

**Steps to reproduce the issue:**
1. Run `qos/test_qos_sai.py::TestQosSai::testQosSaiDwrr` test.
2.
3.

**Describe the results you received:**
The test failed.

**Describe the results you expected:**
Test should pass.

**Additional information you deem important:**

```
SONiC Software Version: SONiC.20220531.40
SONiC OS Version: 11
Distribution: Debian 11.7
Kernel: 5.10.0-18-2-amd64
Build commit: c98dfa6269
Build date: Fri Oct  6 22:15:21 UTC 2023
Built by: cloudtest@b88cfdddc000000
```

**Comment by bingwang-ms** [comment]
@JibinBao Can you please help take a look at this issue? Thanks

**Comment by JibinBao** [comment]
will check it

**Comment by JibinBao** [comment]
Fixed:https://github.com/sonic-net/sonic-mgmt/pull/10525
Can you help review the PR?

**Comment by bingwang-ms** [comment]
Closed by PR#10525
